File: Rains of Reason/alphabeticShift.py
def solution(inputString):
    outputString = ""
    for inputChar in inputString:
        if inputChar == "z":
            outputString += "a"
            continue
        code = ord(inputChar)
        outputChar = chr(code + 1)
        outputString += outputChar
    return outputString

print(solution("zabc"))

File: Rains of Reason/alphabeticShift.js
/**
 *
 * @param {string} inputString
 */
function solution(inputString) {
  let outputString = "";
  const inputArr = inputString.split("");
  for (let i = 0; i < inputArr.length; i++) {
    const inputChar = inputArr[i];
    if (inputChar == "z") {
      outputString += "a";
      continue;
    }
    const code = inputChar.charCodeAt();
    const outputChar = String.fromCharCode(code + 1);
    outputString += outputChar;
  }
  return outputString;
}

console.log(solution("zabc"));
